src/main/java/pl/byczazagroda/trackexpensesappbackend/dto/FinancialTransactionDTO.java

```diff
@@ -1,10 +1,11 @@
 package pl.byczazagroda.trackexpensesappbackend.dto;
 
+import com.fasterxml.jackson.annotation.JsonInclude;
 import pl.byczazagroda.trackexpensesappbackend.model.FinancialTransactionType;
 
 import java.math.BigDecimal;
 import java.time.Instant;
 
-public record FinancialTransactionDTO(Long id, BigDecimal amount, String description,
-                                      FinancialTransactionType type, Instant date, Long categoryId) {
+public record FinancialTransactionDTO(Long id, BigDecimal amount, @JsonInclude String description,
+                                      FinancialTransactionType type, Instant date, @JsonInclude Long categoryId) {
```

Comment:
`@JsonInclude` -> `@JsonInclude(JsonInclude.Include.ALWAYS)` 
That should work you can check the discord fo further info